### PR TITLE
DAH-1113 Support all statuses for the application sidebar component

### PR DESCRIPTION
**Pull request by christine-sfg**
DAH-1113

Supporting two states for the application deadline component on the details page side bar - open (before application deadline) and closed (after application deadline). This should be an improvement over the existing angular component which has hardcoded the english 'at' into the date time

image

image

### CODE REVIEW

File: app/javascript/pages/listings/listing-detail.tsx [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
 import { getReservedCommunityType } from "../../util/languageUtil"
-import {
-  RailsListing,
-  getListingImageCardStatuses,
-  getListingAddressString,
-} from "../../modules/listings/SharedHelpers"
+import { getListingAddressString, RailsListing } from "../../modules/listings/SharedHelpers"
 import { ListingEvent } from "../../api/types/rails/listings/BaseRailsListing"
 
+const alertIconColor = "#E31C3D"
```
Comment (christine-sfg) [inline app/javascript/pages/listings/listing-detail.tsx:33]: I'm new to tailwind. Would appreciate any help on how to access the configured palette in react.

Comment (jacksongonzales) [reply app/javascript/pages/listings/listing-detail.tsx:33]: [here's](https://github.com/bloom-housing/bloom/blob/dev/ui-components/tailwind.config.js#L35-L75) the color tokens from the bloom theme. I don't see `#E31C3D` in there, though the difference between that and `red.700` (`#E41D3D`) is negligible. if it's fine to roll with that, I think you'd wanna do something like
```suggestion
import bloomTheme from "../../../../tailwind.config"
...
iconColor={!isApplicationOpen && bloomTheme.theme.colors.red.700}
...
```

**Review by jacksongonzales — APPROVED** [review]
looks good! lgtm

File: app/javascript/pages/listings/listing-detail.tsx [hunk 2]
```diff
@@ -102,7 +102,19 @@ const ListingDetail = () => {
       >
         <aside className="w-full static md:absolute md:right-0 md:w-1/3 md:top-0 sm:w-2/3 md:ml-2 h-full md:border border-solid bg-white">
           <div className="hidden md:block">
-            <ApplicationStatus {...getListingImageCardStatuses(listing, false)[0]} />
```
Comment (emilyjablonski) [inline app/javascript/pages/listings/listing-detail.tsx:105]: Now that we're able to work in bite-size pieces do you want to add unit tests for this in this PR?

Comment (christine-sfg) [reply app/javascript/pages/listings/listing-detail.tsx:105]: Great question. I don't think this component is currently setup well for a unit test. I'll address that in DAH-1122

**Pull request merged**